apps/src/templates/teacherDashboard/AssignmentSelector.jsx

```diff
@@ -323,12 +336,11 @@ export default class AssignmentSelector extends Component {
             </div>
             <select
               id="uitest-secondary-assignment"
-              value={selectedSecondaryId}
+              value={secondaryId}
```

Comment:
this could end up being undefined (if there are no `scriptAssignIds`) or null. does the right thing happen in both cases?